docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,72 +1,304 @@
 [role="xpack"]
 [[canvas-workpad]]
+== Creating and building a workpad
+
+To create a dynamic, multi-page, pixel perfect presentation, create and build your Canvas workpad.
+
+In need of some workpad inspiration? Check out the link:https://www.elastic.co/blog/[Elastic Blog]. 
+
+[float]
+[[create-canvas-workpad]]
 === Create a workpad
 
 When you create a workpad, you'll start with a blank page, or you can choose a workpad that is already started for you. 
 
 . Open *Canvas*.
 
-. Choose one of the following options:
+. On the *Canvas workpads* page, choose one of the following options:
 
 * To start with a single, blank workpad, click *Create workpad*.
 
-* To create a workpad from a template, select *Templates*, then select a preconfigured template.
+* To create a workpad from a template, select *Templates*, then click on a preconfigured template.
 
 * To import an existing workpad, click and drag a workpad JSON file to the *Import workpad JSON file* field.
 
-For advanced workpad examples, add a <<add-sample-data, sample Kibana data set>>, then select *Canvas* from the *View Data* dropdown list.
-
-For more workpad inspiration, go to the link:https://www.elastic.co/blog/[Elastic Blog]. 
+* To use a sample data workpad, click *Try the sample data workpads*, add the data, then select *Canvas* from the *View Data* dropdown list.
 
 [float]
-=== Personalize your workpad
+[[customize-workpad-basic]]
+=== Customize your workpad
 
-To personalize your workpad, use the editor on the right.
+To make your workpad look the way you want, use the editor on the right.
 
-. Add a *Name* to your workpad, for example, `My Canvas Workpad`.
+. Add a *Name* to your workpad. For example, `My Canvas Workpad`.
 
-. In the *Width* and *Height* fields, specify the size of your workpad. 
+. In the *Width* and *Height* fields, specify the size. 
 
-. Select the layout for your workpad. 
+. Select the layout. 
 +
 For example, click *720p* for a traditional presentation layout.
 
 . Click the *Background color* picker, then select the background color for your workpad.
 +
+[role="screenshot"]
 image::images/canvas-background-color-picker.gif[]
 
-. If you already know that you want to apply styles to the entire workpad, use the *Global CCS overrides*.
+. Apply a set of styles to the entire workpad.
 
 .. Expand *Global CCS overrides*.
 
 .. Enter the CCS, then click *Apply stylesheet*.
 
 [float]
-=== Share your workpad
+[[element-intro]]
+== Showcase your data with elements
 
-When you are ready to share your workpad, create a PDF, or export your workpad to another author.
+Elements are the building blocks of your workpad. You can add and combine elements to create dynamic displays of your data.
 
 [float]
-==== Create a PDF of your workpad
+[[add-canvas-element]]
+=== Add elements
+
+Choose the elements that best display your brand and data.
```

Comment:
I don't feel this line is necessary.  How about removing it and making the second sentence under the intro:  You can add and combine elements to best display your brand and data."   I think "dynamic" is overused.